doc/libf3d/OPTIONS.md

```diff
@@ -43,9 +43,9 @@ Option|Type<br>Default<br>Trigger|Description|F3D option
 :---:|:---:|:---|:---:
 model.matcap.texture|string<br>optional<br>render|Path to a texture file containing a material capture. All other model options for surfaces are ignored if this is set. Model specified by default.|\-\-texture-matcap
 model.color.opacity|double<br>optional<br>render|Set *opacity* on the geometry. Usually used with Depth Peeling option. Multiplied with the `model.color.texture` when present. Model specified by default.|\-\-opacity
-model.color.rgb|vector\<double\><br>optional<br>render|Set a *color* on the geometry. Multiplied with the `model.color.texture` when present. Model specified by default.|\-\-color
+model.color.rgb|double_vector<br>optional<br>render|Set a *color* on the geometry. Multiplied with the `model.color.texture` when present. Model specified by default.|\-\-color
```

Comment:
Im not sure this is the right syntax, wdyt @Meakk ?

Reply:
Wdym?

Reply:
Should we write `double_vector` or `vector<double>` in the user doc ?

Reply:
`double[3]`?

Reply:
nope, that would be incorrect, it can receive any number of values.

Reply:
also it even more programmation oriented, which our f3d users are not